### PR TITLE
Add universal save system

### PR DESCRIPTION
**Pull request by derkanw**

### CODE REVIEW

File: Assets/Scripts/Tokens.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using UnityEngine;
+
+public static class Tokens
+{
+    public static readonly string StatsPoints = "PointsForStats";
+    public static readonly string Stats = "PlayerStats";
+    public static readonly string PlayerLevel = "PlayerLevel";
+    public static readonly string Points = "PointsForNewLevel";
+    public static readonly string GameLevel = "GameLevel";
+    public static readonly string EnemyIndex = "EnemyIndex";
```
Comment (pushkinman) [inline Assets/Scripts/Tokens.cs:12]:
```suggestion
    public static const string StatsPoints = "PointsForStats";
    public static const string Stats = "PlayerStats";
    public static const string PlayerLevel = "PlayerLevel";
    public static const string Points = "PointsForNewLevel";
    public static const string GameLevel = "GameLevel";
    public static const string EnemyIndex = "EnemyIndex";
```

File: Assets/Scripts/UI/ButtonManager.cs [hunk 2]
```diff
@@ -61,8 +61,7 @@ public void OnStatsButton()
 
     public void OnResetButton()
     {
-        PlayerPrefs.DeleteAll();
-        File.Delete(SaveSystem.path);
+        SaveSystem.DeleteAll();
         SceneManager.LoadScene("Scenes/MainLevel");
```
Comment (pushkinman) [inline Assets/Scripts/UI/ButtonManager.cs:65]: "Scenes/MainLevel" put the value in a separate field, do not hardcode

**Review by pushkinman — CHANGES_REQUESTED** [review]
Small fixes

File: Assets/Scripts/UI/ButtonManager.cs (hunk 2 above)
Comment (derkanw) [reply Assets/Scripts/UI/ButtonManager.cs]: Fixed

**Pull request merged**